core/src/main/java/org/elasticsearch/common/settings/SettingsModule.java

```diff
@@ -218,4 +220,8 @@ public IndexScopedSettings getIndexScopedSettings() {
     public ClusterSettings getClusterSettings() {
         return clusterSettings;
     }
+
+    public SettingsFilter getSettingsFilter() {
```

Comment:
Exposed to ActionModule to build rest handlers. Still needs to be bound I think.